src/practices/policy.yaml

```diff
@@ -1,7 +1,7 @@
 osVersion:
   darwin:
-    # High Sierra
-    ok: ">=10.14.4"
+    # Latest High Sierra or Catalina
+    ok: "<10.15 >=10.14.6 || >=10.15.1"
     # Sierra
     nudge: ">=10.12.6"
```

Comment:
Would it make sense to bump this to High Sierra?

Reply:
Definitely open to bumping; however, I've been basing the lowest supported version of macOS on whether or not security updates are still being released (in other words, still officially supported by Apple). 
Looks like the last Sierra push was in September: https://support.apple.com/en-us/HT201222

Reply:
ah, good to know!